Sources/NIO/SelectableEventLoop.swift

```diff
@@ -136,7 +136,7 @@ internal final class SelectableEventLoop: EventLoop {
     }
 
     /// Is this `SelectableEventLoop` still open (ie. not shutting down or shut down)
-    internal var isOpen: Bool {
+    internal var acceptingRegistrations: Bool {
```

Comment:
This rename isn't strictly necessary anymore, but I thought it was a better name. Can change it back if preferred.

Reply:
I'm fine with this, maybe you should expand the comment a little though.